src/Microsoft.Azure.SignalR.Common/ServiceCredential.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalR.Common
+{
+    public class ServiceCredential
```

Comment:
>ServiceCredential [](start = 17, length = 17)

no need to expose such a public class to user?

Reply:
Will be used in SDK `SignalRService`, it need to be public.

Reply:
Please use InternalVisibleTo

Reply:
Ok, I have made the class internal